std/random.d

```diff
@@ -527,16 +528,17 @@ alias MinstdRand = LinearCongruentialEngine!(uint, 48271, 0, 2147483647);
 The $(LUCKY Mersenne Twister) generator.
  */
 struct MersenneTwisterEngine(UIntType, size_t w, size_t n, size_t m, size_t r,
-                             UIntType a, size_t u, size_t s,
+                             UIntType a, size_t u, UIntType d, size_t s,
                              UIntType b, size_t t,
-                             UIntType c, size_t l)
+                             UIntType c, size_t l, UIntType f)
```

Comment:
Yes, technically it's a breaking change. I'd consider it quite unlikely that anyone is using the template directly, though (and if they are it is probably not safe to do so given the missing parameters).

I'll make sure this is noted prominently in the changelog, though.